docker/docker-entrypoint.sh

```diff
@@ -36,5 +36,6 @@ node --max-old-space-size=$NODE_SPACE \
 	/usr/local/lib/node_modules/@subql/node/bin/run $@ \
 	--disable-historical=true
 	--timestamp-field
+	--db-schema=public
```

Comment:
I think you should also add a `\` at the end to ensure continuity or write those in one line 
```suggestion
	--disable-historical=true \
	--timestamp-field \
	--db-schema=public
```